sklearn/metrics/scorer.py

```diff
@@ -282,6 +282,80 @@ def check_scoring(estimator, scoring=None, allow_none=False):
             "have a 'score' method. The estimator %r does not." % estimator)
 
 
+def check_multimetric_scoring(estimator, scoring=None, allow_none=False):
```

Comment:
I'm inclined to make this private for a release or two, say until we've worked out what we'll do about using `precision_recall_fscore` to return 3 metrics for each class.